src/aat/resources/features/F-026/F-026.feature

```diff
@@ -0,0 +1,58 @@
+@F-026
+Feature: F-026: Get a case data with UI layout
+
+  Background: Load test data for the scenario
+    Given an appropriate test context as detailed in the test data source
+
+  @S-078
+  Scenario: must return the list of cases and status code 200 for correct inputs
+    Given a case that has just been created as in [Standard_Full_Case_Creation_Data]
```

Comment:
Why create a case at all? Totally non sense.

Reply:
My assumption: A new case created, because when these tests run on any environment, it should have at least one case to return.
However as per your explanation, assuming cases are already available in our environments, removed this case creation step from tests.